packages/lab/src/Flow/Node/Node.tsx

```diff
@@ -62,6 +65,8 @@ export interface HvFlowNodeProps<T = any> extends HvFlowBaseNodeProps<T> {
   labels?: HvFlowBaseNodeProps["labels"] & Partial<typeof DEFAULT_LABELS>;
   /** A Jss Object used to override or extend the styles applied to the component. */
   classes?: HvFlowNodeClasses;
+  /** Remove the ability to customize the label of the Node */
+  disableInlineEdit?: boolean;
 }
 
 export const HvFlowNode = ({
```

Comment:
We need to run Chromatic by adding the `visual` tag to the PR as there will be changes 🙏